src/Test/contconic.jl

```diff
@@ -1144,6 +1144,90 @@ exptests = Dict("exp1v" => exp1vtest,
 
 @moitestset exp
 
+function _pow1test(model::MOI.ModelLike, config::TestConfig, vecofvars::Bool)
+    atol = config.atol
+    rtol = config.rtol
+    # Problem POW1
+    # max z
+    #  st  x^0.9 * y^(0.1) >= |z| (i.e (x, y, z) are in the 3d power cone with a=0.9)
+    #      x == 2
+    #      y == 1
+    a = 0.9
+    @test MOIU.supports_default_copy_to(model, #=copy_names=#false)
+    @test MOI.supports(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}())
+    @test MOI.supports(model, MOI.ObjectiveSense())
+
+    if vecofvars
+        @test MOI.supports_constraint(model, MOI.VectorOfVariables, MOI.PowerCone{Float64})
```

Comment:
I am not sure about this line? Should it be `MOI.PowerCone` or `MOI.PowerCone{Float64}`? Either way it seems to produce different errors.

Reply:
It should be `PowerCone{typeof(a)}` so `PowerCone{Float64}`

Reply:
It seems you need to add `MOI.PowerCone,` here: https://github.com/JuliaOpt/MathOptInterface.jl/blob/ef83d2381b2e26a3e6466b8bc34b6687c7c20c61/test/Bridges/lazy_bridge_optimizer.jl#L17

Reply:
This doesn't seem to solve it. There are some errors in the Bridges unit tests related to the fact that PowerCone is typed. I am not really sure, what's going on there.

Reply:
Can you post the errors or push the suggested fix so that I see the errors on Travis ?